### ISSUE TITLE
LightStep TracingService does not work (most likely http2 issue)

### ISSUE THREAD

**Issue by danielbryantuk** [datawire/ambassador#796]
**Describe the bug**
The TracingService does not currently support LightStep integration

**To Reproduce**
I've created a proof of concept application here, which should deploy a modified OpenTracing Java Microdonuts application into Kubernetes with LightStep distributed tracing integration
https://github.com/danielbryantuk/java-opentracing-walkthrough/tree/master/kubernetes-ambassador-lightstep

However, if you kubectl apply all of the files in this directory you will notice that Ambassador fails to route traffic to the LightStep SaaS API endpoint

Upon debugging Ambassador during startup, this error is found: 

[2018-09-12 16:22:39.658][121][critical][main] source/server/config_validation/server.cc:48] error initializing configuration '/ambassador/ambassador-config-9-envoy.json': cluster_ext_tracing collector cluster must support http2 for gRPC calls

Looking at the generated Envoy JSON, we can see that the 'cluster_ext_tracing' doesn't have http2 enabled:

       {
        "name": "cluster_ext_tracing",
        "connect_timeout_ms": 3000,
        "type": "strict_dns",
        "lb_type": "round_robin",        
        "hosts": [
          {
            "url": "tcp://collector-grpc.lightstep.com:443"
          }
          
        ]},

**Expected behavior**
LightStep tracing to work :-)

**Versions (please complete the following information):**
 - Ambassador: 0.39.0
 - Kubernetes environment: Google Kubernetes Engine
 - Version: v1.9.7-gke.6

**Additional context**

**Comment by danielbryantuk** [comment]
Presumedly this will be a fix in the config.py file, with the module_config_tracing function:

https://github.com/datawire/ambassador/blob/33df0cd9e445fbbe9ce32636083cc7501d5b7a64/ambassador/ambassador/config.py#L1621

**Comment by danielbryantuk** [comment]
I've found this Envoy/LightStep example, which appears to confirm out suspicions: https://github.com/rnburn/envoy-lightstep-example/blob/master/service-envoy-1.json

```
{
        "name": "lightstep",
        "connect_timeout_ms": 1000,
        "type": "strict_dns",
        "lb_type": "round_robin",
        "features": "http2",
        "ssl_context": {
          "ca_cert_file": "/etc/ssl/certs/ca-certificates.crt",
          "verify_subject_alt_name": ["collector-grpc.lightstep.com"]
        },
        "hosts": [
          {
            "url": "tcp://collector-grpc.lightstep.com:443"
          }
        ]
      }
```

**Comment by danielbryantuk** [comment]
It's worth mentioning that I haven't included a LightStep API key in the Kubernetes config in the example (within the lightstep-config.yaml), and so whoever does work on this will need to get a test account from the LightStep team

**Comment by richarddli** [comment]
Fixed in 0.40
